README.md

```diff
@@ -38,6 +38,8 @@
 * iOS 9.0+
 * Xcode 7.3+
 
+> **Note:** Branches for Swift 2.3 ([`swift-2.3`](https://github.com/Lickability/PinpointKit/tree/swift-2.3)) and 3.0 ([`swift-3.0`](https://github.com/Lickability/PinpointKit/tree/swift-3.0)) are being kept up to date as Xcode 8 betas are released.
```

Comment:
>up to date
should be "up-to-date" according to https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwiyrs7w_4fOAhUG9R4KHdIJAbwQFggmMAE&url=http%3A%2F%2Fenglish.stackexchange.com%2Fquestions%2F180611%2Fdo-i-keep-myself-up-to-date-or-up-to-date-on-something&usg=AFQjCNHOZ5dAlKJdAmWH2zp3PrCMMTKO2w&sig2=-6ii4_W2AQ6HSKWsKCJurw&bvm=bv.127521224,d.dmo